config/app/manifest/en_US.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Why does this file exist here? If it's created by your test case then you don't have the virtual filesystem configured correctly.

Reply:
This file is created by new manifest console command which we have run on root directory

Reply:
It MUST not exist. This tool will create those files but will not have any of those files. I guess that either you created it by running the tool inside the tool or that your test created it.

These files MUST be removed and the tests should only deal with files in the virtual filesystem, no real file should be written by this tool.